fiftyone/operators/builtin.py

```diff
@@ -1885,38 +1929,52 @@ def config(self):
     def resolve_input(self, ctx):
         inputs = types.Object()
 
-        saved_views = ctx.dataset.list_saved_views()
-
-        if saved_views:
-            saved_view_selector = types.AutocompleteView()
-            for key in saved_views:
-                saved_view_selector.add_choice(key, label=key)
-
-            inputs.enum(
-                "name",
-                saved_view_selector.values(),
-                default=None,
-                required=True,
-                label="Saved view",
-                description="The saved view to delete",
-                view=saved_view_selector,
-            )
-        else:
-            prop = inputs.str(
-                "msg",
-                label="This dataset has no saved views",
-                view=types.Warning(),
-            )
-            prop.invalid = True
+        _delete_saved_view_inputs(ctx, inputs)
 
         return types.Property(
             inputs, view=types.View(label="Delete saved view")
         )
 
     def execute(self, ctx):
-        name = ctx.params["name"]
+        names = _to_string_list(ctx.params["names"])
+
+        for name in names:
+            ctx.dataset.delete_saved_view(name)
+
+
+def _delete_saved_view_inputs(ctx, inputs):
+    saved_views = ctx.dataset.list_saved_views()
+
+    if not saved_views:
+        prop = inputs.str(
+            "msg",
+            label="This dataset has no saved views",
+            view=types.Warning(),
+        )
+        prop.invalid = True
+        return
+
+    names = _to_string_list(ctx.params.get("names", []))
+
+    saved_view_selector = types.AutocompleteView(multiple=True)
```

Comment:
Ideally can make this `DropdownView(multiple=True)` to avoid manually enforcing valid values below